src/map/utils/battleutils.cpp

```diff
@@ -2554,8 +2543,38 @@ namespace battleutils
             {
                 num += 1;
             }
+            
+            return num;
+        }
+        
+        return 0;
+    }
+    
+    /************************************************************************
+    *                                                                       *
+    *  Returns multihits for special mobs / pets 							*
+    *                                                                       *
+    ************************************************************************/
+    
+    uint8 CheckSpecialMobMultiHits(CBattleEntity* PEntity)
+    {
+        if (PEntity->objtype == TYPE_MOB || PEntity->objtype == TYPE_PET)
+        {
+            uint8 num = 1;
+            
+            // Check for unique mobs
+            switch (PEntity->id)
+            {
+                case 17498522: // Charybdis 2-6
```

Comment:
MOBMOD_MULTIHIT or something
don't want this to be affected by id shifts (and i just hate hardcoded id checks in general)